.github/actions/npm-build-project/action.yml

```diff
@@ -19,10 +22,14 @@ runs:
       with:
         node-version: ${{ inputs.node_version }}
         cache: 'npm'
-
+        registry-url: 'https://npm.pkg.github.com'
+        scope: '@ukhomeoffice'
+        
     - name: Install npm dependencies
       run: npm --loglevel warn ${{ inputs.install_command || 'ci' }}
```

Comment:
Following this: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#use-private-packages

Do we need to add the following flag? `--ignore-scripts`